### DISCUSSION TITLE
map last modified - date and time in specific node

### DISCUSSION THREAD

**Discussion by GiovanniVG** [freeplane/freeplane#690] [Help]
I was looking for a solution to show the date / time of the last modification of any not in the entire map in one specific node. It took me some hours to go through different solution posts, the lower mentioned was worked for me perfectly. 
Most of you may already know this, but maybe this post will help one "non-power-user" in the future... I hope.

Text in node:
`= "map last modified: " + format(c.findAll().collect{ it.lastModifiedAt }.max(),"yyyy-MM-dd " + "/ " +"HH:mm:ss")`
_note:
"scripts exucation" and "Permit File/Read Operations" must be enabled in preferences!_

**Comment by EdoFro** [comment]
just a note:
there are some kind of node's modifications that do not update its "last modified date". I don't know exactly all cases, but actions like  moving a node to another position, deleting its children or copy/pasting nodes won't change the "last modified date" from any node.

**Reply by GiovanniVG** [reply]
Thanks for the note, that´s correct and I know this. 
Do you have any idea or easy solution (formula?) how to show the "real" last modified timestamp (inkl. node-moving changes)?

**Reply by euu2021** [reply]
Maybe, the Listeners capture more kinds of changes?

**Reply by EdoFro** [reply]
> Do you have any idea or easy solution (formula?) how to show the "real" last modified timestamp (inkl. node-moving changes)?

sorry, I don't.

**Reply by euu2021** [reply]
Using a Script in the Conditional Styles may be another idea. See #520 to learn how to use it.
